tests/e2e/runner/__tests__/minter-suite-v2/i-shared-minter-v0.test.ts

```diff
@@ -119,17 +129,59 @@ describe("iSharedMinterV0 event handling", () => {
     });
   });
 
-  // TODO - need ERC20 minter to test event handling of "ProjectCurrencyInfoUpdated"
-  // ERC20 minter is not yet developed at the time of writing this test
-  // describe("ProjectCurrencyInfoUpdated", () => {
-  //   afterEach(async () => {
-  //     // TODO
-  //   });
-
-  //   test("Currency is updated and configured", async () => {
-  //     // TODO - need ERC20 minter to test this
-  //   });
-  // });
+  describe("ProjectCurrencyInfoUpdated", () => {
+    afterEach(async () => {
+      // clear the minter for project zero
+      try {
+        await sharedMinterFilterContract
+          .connect(artist)
+          .removeMinterForProject(0, genArt721CoreAddress);
+      } catch (error) {
+        // swallow error in case of test failure
```

Comment:
@ryley-o why do we just swallow this error?

Reply:
If the test failed for some reason, project 0 might not have a minter assigned, and the try block here might fail. Ideally, we will only see the failed test, and won't also see more information about a failed call in the afterEach.

Reply:
yes - updated with more comments and cleaned up some of the try/catch blocks across all of our e2e tests in [2d0cd26](https://github.com/ArtBlocks/artblocks-subgraph/pull/235/commits/2d0cd260b529c307c6e10beb4abdb882d1fa04f1)